pyproject.toml

```diff
@@ -0,0 +1,45 @@
+# With this the user can use a custom
+# python environment, like those created
+# with venv or pyenv. The deault will be
+# the system python.
+[virtualenvs]
+prefer-active-python = true
+
+# This isnt really needed on pratical aspect
+# since this is only for organized the functional tests
+# but is necessary since its a requirement for poetry
+[tool.poetry]
+name = "floresta-functional-tests"
+version = "0.0.1"
+description = "collection of tools to help with functional testing of Floresta"
+authors = [
+  "lordshashank",
+  "qlrd <qlrddev@gmail.com>"
+]
+
+# Replace requirements.txt to here
```

Comment:
Can we have both? I think some people might want to run manually, we should give this option (but discourage)

Reply:
Well, you can make the supposition that, if the guy wants to run things outside from pyenv, he knows how to change the version and is seeking for more control over this tests.

IMO, you could just specify a version in the README that matches the configuration that ur working on.

Reply:
maybe add something like this on README can be a "middle-way"?

```bash
python -m venv $HOME/.floresta-test
source $HOME/.floresta-test/bin/activate

# now you're in virtualenv
pip install -r tests/requirements.txt
python tests/run_tests.py
```

Reply:
> Since python system can differ from user to another, this can lead, I believe, to inconsistencies if, for example, Alice uses py3.9 and Bob uses py3.13 (so the same test can work for Alice and broke for Bob).

Usually, python won't break with forward versions (i.e. a test that runs on 3.9 will almost always work on 3.11, but the opposite isn't necessarily true). We can use the same trick as rust versions. Pick a recent-ish version and make sure our tests runs on that. If it breaks in older version, it's not our problem :joy:

Reply:
> If it breaks in older version, it's not our problem

Fair enough 😆 

> you could just specify a version

Fixed (defined it as >=3.12 in `README` and `pyproject.toml`)